Various spelling tweaks and small changes in beta release notes (#17641)
home-assistant/home-assistant.github.io: source/_posts/2021-05-05-release-20215.markdown

{% details "Shelly" %}

> The Shelly Wifi RSSI sensor unit was wrong, it is **dBm** and not **dB**.

Reviewer: This is actually incorrect.

```suggestion
The Shelly Wi-Fi RSSI sensor unit was wrong, it is **dBm** and not **dB**.
```

See:
- <https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/wi-fi>
- <https://developers.google.com/style/word-list#letter-w>

For now, we have a fallback ENV variable `USING_UDEV=1` which you can set on your container
to get the old behavier and give you time for using the docker way to map/use devices.

> ([@pvizeli]- [#180])

Reviewer: ```suggestion
([@pvizeli]- [#180])

[#180]: https://github.com/home-assistant/docker/pull/180
```

{% enddetails %}

> [#180]: https://github.com/home-assistant/docker/pull/180

Reviewer: This list is generated :)
```suggestion
```

Author: Cool, didn't know that!
